packages/solana/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { PublicKey, type Transaction } from "@solana/web3.js";
+import { TurnkeyActivityError, TurnkeyClient } from "@turnkey/http";
+
+export class TurnkeySigner {
+  public readonly organizationId: string;
+  public readonly client: TurnkeyClient;
+
+  constructor(input: { organizationId: string; client: TurnkeyClient }) {
+    this.organizationId = input.organizationId;
+    this.client = input.client;
+  }
+
+  /**
+   * This function takes a Solana transaction and adds a signature with Turnkey
+   *
+   * @param tx Transaction object (native @solana/web3.js type)
+   * @param fromAddress Solana address (base58 encoded)
+   */
+  public async addSignature(tx: Transaction, fromAddress: string) {
```

Comment:
supporting just `addSignature` makes sense given this is all a user needs support for, at least for now. (when we implement a signer interface, then things might change as we support `sign` and `partialSign`, etc)

Reply:
Yeah definitely agree. We can extend the interface as needed in the future. There's also the "sign message" functionality; this can be added fairly easily but I've left it out for now since it's not been requested just yet.